pynest/nest/server/hl_api_server.py

```diff
@@ -216,8 +216,7 @@ def do_exec(args, kwargs):
                 data[variable] = locals_.get(variable, None)
         else:
             data = locals_.get(kwargs["return"], None)
-
-            response["data"] = get_or_error(nest.serialize_data)(data)
+        response["data"] = get_or_error(nest.serialize_data)(data)
```

Comment:
@babsey For better readability, I would suggest to leave the empty line after the `data = ...` line. And you could turn the for loop into a dict comprehension above:

```python
data = {variable: locals_.get(variable, None) for variable in kwargs["return"]}
```